test/todo.spec.js

```diff
@@ -18,21 +18,21 @@ describe('change attr', () => {
   });
 
   test('it change the date when given "date" as an argument and a new date', () => {
-    myTodo.changeAttr('date', '2021-05-11');
-    expect(myTodo.date).toBe('2021-05-11');
+    myTodo.changeAttr('date', '2021-05-28');
+    expect(myTodo.date).toBe('2021-05-28');
   });
 });
 
-const myTodo = new Todo('myTodoTitle', 'myDescription', '2021-05-11', 1, 0);
+const myTodo = new Todo('myTodoTitle', 'myDescription', '2021-05-17', 1, 0);
 
 describe('find Props', () => {
   test('it returns an array with the instance values', () => {
-    expect(myTodo.findProps()).toEqual(['myTodoTitle', 'myDescription', '2021-05-11', 1, 0, 1, false]);
+    expect(myTodo.findProps()).toEqual(['myTodoTitle', 'myDescription', '2021-05-17', 1, 0, 1, false]);
   });
 });
 
 describe('format Props', () => {
   test('it returns an array with the instance values formatted', () => {
-    expect(myTodo.formatProps()).toEqual(['myTodoTitle', 'myDescription', '2021-05-11', 'Medium', 'myTitle']);
+    expect(myTodo.formatProps()).toEqual(['myTodoTitle', 'myDescription', '2021-05-17', 'Medium', 'myTitle']);
   });
-});
\ No newline at end of file
+});
```

Comment:
We need to write test for the following methods;
1. `addToProject`
2. `editTask`